packages/client/modules/userDashboard/components/OrgBilling/OrgPlanDrawer.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import {commitLocalUpdate, useFragment} from 'react-relay'
+import {OrgPlanDrawer_organization$key} from '../../../../__generated__/OrgPlanDrawer_organization.graphql'
+import ResponsiveDashSidebar from '../../../../components/ResponsiveDashSidebar'
+import {Close} from '@mui/icons-material'
+import {PALETTE} from '../../../../styles/paletteV3'
+import {BezierCurve, Breakpoint, DiscussionThreadEnum, ZIndex} from '../../../../types/constEnums'
+import LabelHeading from '../../../../components/LabelHeading/LabelHeading'
+import {desktopSidebarShadow} from '../../../../styles/elevation'
+import useBreakpoint from '../../../../hooks/useBreakpoint'
+import PlainButton from '../../../../components/PlainButton/PlainButton'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import {ICON_SIZE} from '../../../../styles/typographyV2'
+import OrgPlanDrawerContent from './OrgPlanDrawerContent'
+
+const DrawerHeader = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  justifyContent: 'space-between',
+  padding: '16px 8px 16px 16px',
+  width: '100%'
+})
+
+const Drawer = styled('div')<{isDesktop: boolean; isOpen: boolean}>(({isDesktop, isOpen}) => ({
+  boxShadow: isDesktop ? desktopSidebarShadow : undefined,
+  backgroundColor: '#FFFFFF',
+  display: 'flex',
+  flex: 1,
+  flexDirection: 'column',
+  justifyContent: 'stretch',
+  overflow: 'hidden',
+  position: isDesktop ? 'fixed' : 'static',
+  bottom: 0,
+  top: 0,
+  right: isDesktop ? 0 : undefined,
+  transition: `all 200ms ${BezierCurve.DECELERATE}`,
+  userSelect: 'none',
+  transform: `translateX(${isOpen ? 0 : DiscussionThreadEnum.WIDTH}px)`,
+  width: DiscussionThreadEnum.WIDTH,
+  zIndex: ZIndex.SIDEBAR,
+  height: '100%',
+  '@supports (height: 1svh) and (height: 1lvh)': {
+    height: isDesktop ? '100lvh' : '100svh'
+  }
+}))
+
+const CloseIcon = styled(Close)({
+  color: PALETTE.SLATE_600,
+  cursor: 'pointer',
+  '&:hover': {
+    opacity: 0.5
+  }
+})
+
+const StyledCloseButton = styled(PlainButton)({
+  height: ICON_SIZE.MD18,
+  display: 'flex',
+  alignItems: 'center'
+})
+
+const StyledLabelHeading = styled(LabelHeading)({
+  fontSize: 12,
+  lineHeight: '18px',
+  textTransform: 'none'
+})
+
+type Props = {
+  organizationRef: OrgPlanDrawer_organization$key
+}
+
+const OrgPlanDrawer = (props: Props) => {
+  const {organizationRef} = props
+  const organization = useFragment(
+    graphql`
+      fragment OrgPlanDrawer_organization on Organization {
+        id
+        showDrawer
+        tier
+      }
+    `,
+    organizationRef
+  )
+  const {id: orgId, tier, showDrawer} = organization
+  const atmosphere = useAtmosphere()
+  const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
+
+  const toggleSidebar = () => {
+    commitLocalUpdate(atmosphere, (store) => {
+      const org = store.get(orgId)
+      if (!org) return
+      const showDrawer = org.getValue('showDrawer')
```

Comment:
What is the reason for storing this in store rather than state? Feels like adds extra complexity

Reply:
Keeping it in the client schema means we can avoid prop drilling, i.e. passing the `showDrawer` state around a bunch of components. 

It's currently being used in the OrgPlanDrawer and OrgPlansAndBillingHeading components and could be used in more components in the future. If we change it to state, I'd need to pass it through a few components. This is following the same pattern that we have for a few other drawers in the app.